packages/dx-react-scheduler-material-ui/src/templates/views/horizontal/time-table/cell.test.jsx

```diff
@@ -68,28 +74,33 @@ describe('Horizontal view TimeTable', () => {
       .toHaveLength(1);
   });
   it('should call date format function', () => {
-    const formatDate = jest.fn();
-    formatDate.mockImplementation(() => 'time');
     const tree = shallow((
-      <Cell {...defaultProps} formatDate={formatDate} />
+      <Cell {...defaultProps} />
     ));
 
-    expect(formatDate)
+    expect(defaultProps.formatDate)
       .toHaveBeenCalledWith(defaultProps.startDate, { day: 'numeric' });
     expect(tree.find(`.${classes.text}`).props().children)
       .toBeTruthy();
   });
   it('should call date format function with month and day parameter', () => {
-    const formatDate = jest.fn();
-    formatDate.mockImplementation(() => 'time');
     const startDate = new Date(2018, 6, 1, 16);
     const tree = shallow((
-      <Cell formatDate={formatDate} startDate={startDate} />
+      <Cell {...defaultProps} startDate={startDate} />
     ));
 
-    expect(formatDate)
+    expect(defaultProps.formatDate)
       .toHaveBeenCalledWith(startDate, { day: 'numeric', month: 'short' });
     expect(tree.find(`.${classes.text}`).props().children)
       .toBeTruthy();
   });
+  it('shouldn\'t call date format function with month and day parameter if it is the beginning of the month and the today\'s date', () => {
```

Comment:
Should it be `...beginning of the month and the today...` -> `...beginning of the month or the today...` ?